lib/bulk/common.js

```diff
@@ -1252,7 +1235,7 @@ Object.defineProperty(BulkOperationBase.prototype, 'length', {
 module.exports = {
   Batch,
   BulkOperationBase,
-  bson,
+  bson: BSON,
```

Comment:
This looks suspiciously like we should just move the instantiation to the places that are accessing this `common` file. More importantly, everything that needs to access the BSON types directly in our code should be using [retrieveBSON](https://github.com/mongodb/node-mongodb-native/blob/master/lib/utils.js#L810) rather than directly importing `bson`. I know this wasn't the exact scope of the ticket, but I think this is a good opportunity to clear this up.